src/content/en/updates/posts/2016/03/web-enabled-internet-of-things.markdown

```diff
@@ -0,0 +1,545 @@
+
+My demo app is hosted at <https://edison-webbt.appspot.com/> but the
+Eddystone protocol has a limit on the size of the URL (max 18 bytes,
+more info
+[here](https://github.com/google/eddystone/blob/master/eddystone-url/docs/config-service-spec.md#34-uri-data)), 
+so I have to use a URL shortener in order for it to work.
+
+Broadcasting the URL is quite simple. Import the required libraries and
+call a few functions. One way of doing it is calling `advertiseUrl`, when
+the BLE chip is turned on:
+
+{% highlight javascript %}
+var beacon = require("eddystone-beacon");
+var bleno = require('eddystone-beacon/node_modules/bleno');
+
+bleno.on('stateChange', function(state) {    
+  if (state === 'poweredOn') {
+    beacon.advertiseUrl("https://goo.gl/9FomQC", {name: 'Edison'});
+  }   
+}
+{% endhighlight %}
+
+That really couldn't be much easier. Below you see that Chrome finds
+my device nicely.
+
+> NOTE: The Physical Web enabled devices only show up on your phone
+> when Bluetooth is turned on, and you launch Chrome (currently only
+> works with Beta). Additionally, you have to opt into the feature
+> first time you launch Chrome beta.
+
+<img alt="Chrome announces nearby Physical Web beacons" src="/web/updates/images/2016/03/web-enabled-internet-of-things/nearby.png" style="width: 50%; float: left"/>
+<img alt="The web app URL is listed" src="/web/updates/images/2016/03/web-enabled-internet-of-things/physicalweb.png" style="width: 50%; float: left"/>
+<br/>
+
+## Communicating with the sensor/actuator
+
+For talking to my board enhancements I used [Johnny-Five](http://johnny-five.io/).
+In simple cases like this, this is not strictly easier than communicating
+with the raws input (pins) manually, but for bigger projects it can be
+a real help. Johnny-Five has a nice abstraction for talking to the TMP36
+sensor, but for some reason I could only get it to return "undefined"
+as the current temperature so I went ahead and read the temperature
+value manually.
+
+Below you find the simple code for listening to temperature changes
+as well as setting the initial LED color.
+
+{% highlight javascript %}
+var five = require("johnny-five");
+var Edison = require("edison-io");
+var board = new five.Board({
+  io: new Edison()
+});
+
+board.on("ready", function() {
+  var led = new five.Led.RGB({
+      pins: {
+          red: 3,
+          green: 5,
+          blue: 6
+      },
+  });
+
+  colorCharacteristic._led = led;
+  led.color(colorCharacteristic._value);
+  led.intensity(30);
+
+  board.analogRead("A0", function(raw) {
+    var mV = 5 * 1000 * (raw / 1024);
+    var value = (mV / 10) - 50;
+    temperatureCharacteristic.valueChange(value);
+  });
+}
+{% endhighlight %}
+
+Don't worry about the `*Characteristic` variables above. These are
+what I use for communicating via Bluetooth. As you notice, I talk to
+the TMP36 via the analog `A0` port and the voltage legs on the color
+LED cathode are connected to digital pin 3, 5 and 6, which happens
```

Comment:
Consider just writing `color LED` instead of `color LED cathode`. Might just be my personal preference...